### PR TITLE
Limit number of kicked users, and order by inactive time for reapUsers

### PR DESCRIPTION
**Pull request by Half-Shot**

### CODE REVIEW

File: src/DebugApi.ts [hunk 1]
```diff
@@ -199,13 +199,15 @@ export class DebugApi {
...
+        const limit = query["limit"] !== undefined ? parseInt(query["limit"] as string) : undefined;
         const reason = query["reason"] as string;
         const dry = query["dryrun"] !== undefined && query["dryrun"] !== "false";
         const defaultOnline = (query["defaultOnline"] ?? "true") === "true";
         const excludeRegex = query["excludeRegex"] as string;
         this.ircBridge.connectionReap(
-            msgCb, server, since, reason, dry, defaultOnline, excludeRegex
+            msgCb, server, since, reason, dry, defaultOnline, excludeRegex, limit,
         ).catch((err: Error) => {
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```
Comment (jaller94) [inline src/DebugApi.ts:210]: Why is this all of a sudden required without a change to eslint?

File: src/DebugApi.ts [hunk 2]
```diff
@@ -199,13 +199,15 @@ export class DebugApi {
         };
         const server = query["server"] as string;
         const since = parseInt(query["since"] as string);
+        const limit = query["limit"] !== undefined ? parseInt(query["limit"] as string) : undefined;
```
Comment (jaller94) [inline src/DebugApi.ts:202]: `limit` as in maxKickedUsers?
The term limit could apply to many different restrictions and I wish the naming was more specific.

File: src/DebugApi.ts (hunk 1 above)
Comment (Half-Shot) [reply src/DebugApi.ts:210]: My IDE complained about it, and I submitted to it's demands.

File: src/DebugApi.ts (hunk 2 above)
Comment (Half-Shot) [reply src/DebugApi.ts:202]: hmm, how about targetUserCount ?

Comment (Half-Shot) [reply src/DebugApi.ts:202]: I chose limit specifically because it's easier to type into curl, tbh :p

Comment (Half-Shot) [reply src/DebugApi.ts]: Have settled for `targetCount` which is shorter but more descriptive than `limit`.

**Pull request merged**